src/editors/sharedComponents/TinyMceWidget/hooks.js

```diff
@@ -436,6 +440,15 @@ export const setAssetToStaticUrl = ({ editorValue, lmsEndpointUrl }) => {
     const updatedContent = content.replace(currentSrc, portableUrl);
     content = updatedContent;
   });
+
+  /* istanbul ignore next */
```

Comment:
?

Reply:
This directive will exclude the following from test coverage calculations. It's currently not feasible to test TinyMCE functionality in the test suite as far as I know.